docs/source/format/Columnar.rst

```diff
@@ -765,6 +765,72 @@ application.
 We discuss dictionary encoding as it relates to serialization further
 below.
 
+.. _run-end-encoded-layout:
+
+Run-End Encoded Layout
```

Comment:
Interestingly dictionary encoding isn't listed in that list of physical layouts, and REE is probably closest to that in terms of definitions. Should we also add dictionary encoded layout to this list?

Reply:
Dictionary encoding is a bit different in the specification since it's treated separately and not as a "Type", that's maybe the reason it's not listed there.

Reply:
I added both to that list, let me know what you think of the addition and if you think I should tweak it at all.